File: 1.FDP/2° Competencia/Java/Basicos/Ejercicio4.java
import java.util.Scanner;

public class Ejercicio4 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		//
		System.out.println("Calcular el area de un circulo.\n");
		double Radio, Area;
		System.out.println("Radio: ");
		Radio= scan.nextDouble();
		Area= 3.1416*(Radio*Radio);
		System.out.println("Area= " + Area);
		//
	}
}

File: 1.FDP/2° Competencia/C++/Basicos/Ejercicio4.cpp
#include <iostream>
using namespace std;

int main(){
	cout << "Calcular el area de un circulo.\n";
	double Radio, Area;
	cout << "Radio: ";
	cin >> Radio;
	Area= 3.1416 * (Radio * Radio);
	cout << "Area = " << Area << "\n";
	return 0;
}